frontend/app/index.html

```diff
@@ -64,16 +64,16 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/angular-chart.js/1.1.1/angular-chart.js"></script>
    <!-- <script src="https://rawcdn.githack.com/Orange-OpenSource/angular-swagger-ui/0.5.4/dist/scripts/swagger-ui.js"></script> -->
     <script src="https://rawcdn.githack.com/mmumshad/angular-yamljs/master/dist/angular-yamljs.js"></script>
-    <script src="https://cdn.rawgit.com/asifdxtreme/angular-swagger-ui/565ed3ea/dist/scripts/swagger-ui.js"></script>
+    <script src="https://rawcdn.githack.com/asifdxtreme/angular-swagger-ui/master/dist/scripts/swagger-ui.js"></script>
 
     <script src="https://cdnjs.cloudflare.com/ajax/libs/js-yaml/3.11.0/js-yaml.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/yamljs/0.1.5/yaml.js"></script>
-    <script src="https://cdn.rawgit.com/asifdxtreme/angular-swagger-ui/565ed3ea/dist/scripts/modules/swagger-yaml-parser.min.js"></script>
+    <script src="https://rawcdn.githack.com/asifdxtreme/angular-swagger-ui/master/dist/scripts/modules/swagger-yaml-parser.min.js"></script>
```

Comment:
this is your personal cdn resource?

Reply:
the cdn server is not the personal one, the project has been forked in my own local, I will try to push it upstream to angula-swagger-ui, till then sc can use my fork with public cdn server